nixos/modules/hardware/tuxedo-keyboard.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let 
+  cfg = config.tuxedo-keyboard;
+  tuxedo-keyboard = config.boot.kernelPackages.tuxedo-keyboard;
+in
+  {
+    options.tuxedo-keyboard = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+
+        description = ''
+          Enables the tuxedo-keyboard driver.
+          
+          To configure the driver, pass the options to the <literal>boot.kernelParams</literal> configuration.
```

Comment:
And I guess here should `<option></option>` be used, because it's an option, but I don't think it gets rendered differently anyways